clang/lib/Sema/SemaType.cpp

```diff
@@ -6067,6 +6082,9 @@ namespace {
     void VisitBTFTagAttributedTypeLoc(BTFTagAttributedTypeLoc TL) {
       // nothing
     }
+    void VisitHLSLResourcedAttributedTypeLoc(HLSLAttributedResourceTypeLoc TL) {
```

Comment:
I am unsure if this function is named well.
The name has typically been "HLSLAttributedResourceType" but it's been switched around, and I'm unsure if the d after `Resource` is intended?
Also, how does this function semantically differ from `VisitHLSLAttributedResourceTypeLoc` which is already defined in this file?

Reply:
Good catch! This should not be there. I had originally named the type HLSLResourceAttributesType and this is a remnant of that.